src/Project.hs

```diff
@@ -312,7 +355,7 @@ classifyPullRequest pr = case approval pr of
     Conflicted base _ | base /= baseBranch pr -> PrStatusSpeculativeConflict
     Conflicted _ EmptyRebase -> PrStatusEmptyRebase
     Conflicted _ _  -> PrStatusFailedConflict
-    Integrated _ buildStatus -> case buildStatus of
+    Integrated _ buildStatus -> case latestStatus buildStatus of
       BuildPending     -> PrStatusBuildPending
       BuildStarted url -> PrStatusBuildStarted url
```

Comment:
I suppose that mean we only ever get a CI link to just one of the jobs?

Reply:
Indeed, I was thinking the same but didn't want to scope creep anymore than I already did. I'll create an issue.